magefiles/magefile.go

```diff
@@ -393,6 +393,14 @@ func (ci CI) setRequiredEnvVars() error {
 				envVarPrefix = "IMAGE_CONTROLLER"
 				imageTagSuffix = "image-controller-image"
 				testSuiteLabel = "image-controller"
+			case strings.Contains(jobName, "remote-secret-service"):
+				envVarPrefix = "REMOTE_SECRET_SERVICE"
```

Comment:
the env var prefix differs:`REMOTE_SECRET_SERVICE` vs `REMOTE_SECRET` in [infra-deployments PR](https://github.com/redhat-appstudio/infra-deployments/pull/2287/files#diff-d06ff763d99c81add8827eb9d1f3d99d1ce2ae43eaf235926a0bccfad78c5d2aR84), so it has to be updated here or there

Reply:
updated here. Thanks!